corehq/apps/style/static/style/lib/bootstrap/less/utilities.less

```diff
@@ -28,3 +28,8 @@
 .affix {
   position: fixed;
 }
+
+// TODO: Move this to wherever it belongs
+.one-em-bottom {
```

Comment:
this isn't the right place. `/static/style/less` is where all the less updates go `lib` should not be touched.